README.md

```diff
@@ -1,70 +1,71 @@
-# Getting Started with Create React App
+# BookStore--CMS
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+> 
+[Live Demo available here](https://jocular-praline-d06b4e.netlify.app/)
 
-## Available Scripts
+## Built With
 
-In the project directory, you can run:
+- HTML
+- CSS
+- JavaSript/ES6
+- React&redux
+- Linters
+- NPM
+- Webpack
+- Babel
 
-### `npm start`
+### Prerequisites
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- JavaScript (ES6)
+- React
+- Redux
+- Git
+- GitHub
+- VSCode
+- Linters
+- Fetch API
+- Webpack
+- Babel
+- CSS
+- Sass
+- NPM (Software Packages System)
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+### Install
 
-### `npm test`
+- A code editor (preferably VSCode)
+- A browser
+- Node.js (to run javascript files locally)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Getting Started
 
-### `npm run build`
+To get a local copy up and running follow these simple example steps.
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+- Open git bash on local computer
+- clone the repo with:
+  .
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+```
+git clone  <available once done all the project>
+```
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+- run `npm i` to install all dependencies
+- run `npm run build` or `npm run serve` to bundle the project with webpack, and
+- run `npm start` to launch the application in the browser
 
-### `npm run eject`
+## Author
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+- GitHub: [@alfredbis29](https://github.com/Alfredbis29)
+- Twitter: [@AlfredBisimwa](https://twitter.com/AlfredBisimwa1)
+- LinkedIn: [AlfredBis](https://www.linkedin.com/in/kalumuna-bisimwa-0501a81a8/)
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## 🤝 Contributing
```

Comment:
- [ ] Please add a link to your issues page in the `contributing` section, it will make your README file more professional.